views/templates/panes/executor.pug

```diff
@@ -58,7 +56,9 @@
     .compiler-output
     .execution-output
   .bottom-bar.bg-light
-    button.btn.btn-sm.btn-light.trigger-compilation(title="Trigger compilation")
+    button.btn.btn-sm.btn-light.rerun(title="Rerun")
+      span.fas.fa-circle-play
```

Comment:
I'm ok with this one, but fa-replay could also be an option. Whatever you like :)